docker/docker-compose.ltc.yml

```diff
@@ -29,7 +29,7 @@ services:
         - RPCUSER
         - RPCPASS
         - NETWORK
-        - CHAIN
+        - CHAIN=litecoin
```

Comment:
The idea here would be to add an `export CHAIN=litecoin` command to the example workflow, similar to line 64 in the `docker/README.md` file. But since we don't yet have a section for `litecoin` I think this is fine too.